src/extensions/default/JavaScriptCodeHints/styles/brackets-js-hints.css

```diff
@@ -45,15 +45,15 @@
 
 .brackets-js-hints.literal-hint {
     color: #444; /* dark grey */
-    font-family: SourceCodePro;
+    font-family: @sourceFontFamily;
```

Comment:
This is a `.css` file, not `.less`, so you need to explicitly add these vs. using variables.